.ci/check-links.sh

```diff
@@ -31,9 +31,12 @@ URLS=$(
     | grep -v '{' \
     | grep -v -E '\:[0-9]+$' \
     | grep -v 'auth.docker.io' \
+    | grep -v "community.saturnenterprise.io/" \
     | grep -v 'demo.saturnenterprise.io' \
     | grep -v "https://github.com/saturncloud/docs/" \
     | grep -v "https://github.com/saturncloud/website/" \
+    | grep -v "https://fonts." \
+    | grep -v "https://www.kaggle.com/code/fahd09/eda-of-crime-in-chicago-2005-2016/notebook" \
```

Comment:
I just couldn't figure out what was causing the issue with this one. The link seems fine and it doesn't redirect, so I ended up skipping it just because it doesn't seem worth worrying about.